examples/carousel/carousel-1/carousel-1.html

```diff
@@ -0,0 +1,533 @@
+                    </div>  <!-- .carousel-caption -->
+                  </div>   <!-- .item -->
+
+                  <div class="carousel-item" role="group" aria-roledescription="slide" aria-label="6 of 6" aria-describedby="carousel-label-6">
+
+                    <div class="carousel-image">
+                      <a href="#">
+                        <img src="images/mag800-2__800x600.jpg" alt="Mid-American Gardener panelists on the set" />
+                      </a>
+                    </div>
+
+                    <div class="carousel-caption">
+
+                      <h3>
+                        <a href="#" id="carousel-label-6">
+                          Mid-American Gardener: Thursdays at 7 pm
+                        </a>
+                      </h3>
+
+                      <div class="hidden-xs hidden-sm">
+                        <p>Watch the latest episodes.</p>
+                      </div>
+                    </div>  <!-- .carousel-caption -->
+                  </div>   <!-- .item -->
+
+                </div>
+
+
+              </div> <!-- carousel-inner -->
+
+
+            </section><!-- /.carousel -->
+
+            <div class="col-sm-1"></div>
+
+          </div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+      </section>
+
+      <section>
+        <h2>Accessibility Features</h2>
+        <ul>
+          <li>Automatic rotation of slides in the carousel can be easily controlled by any user.
+            <ul>
+              <li>
+                Hovering the mouse over any carousel content pauses automatic rotation.
+                Automatic rotation resumes when the mouse moves away from the carousel unless another condition, such as keyboard focus,  that prevents rotation has been triggered.
+              </li>
+              <li>
+                Moving keyboard focus to any of the carousel content, including the next and previous slide elements, pauses automatic rotation.
+                Automatic rotation resumes when keyboard focus moves out of the carousel content unles another condition, such as mouse hover,  that prevents rotation has been triggered.
+              </li>
+              <li>The carousel also contains a rotation control button that can stop and start automatic rotation.
+                <ul>
+                  <li>The rotation control button is the first element in the screen reader reading order.</li>
+                  <li>The button is hidden off-screen when it does not have keyboard focus, and becomes visible when users move focus to it with <kbd>Tab</kbd>.</li>
+                  <li>
+                    When not visible, the button is positioned off-screen, instead of being hidden with <code>display:none</code>, so it is always available to screen reader users.
+                    This is an important aspect of supporting users of screen readers that do not move the mouse or keyboard focus as the user is reading.
```

Comment:
What is this line trying to say? It seems a little clunky. Are we saying something like "This is an important aspect of supporting users of screen readers that **the carousel does** not move the mouse or keyboard focus as the user is reading." ?

Reply:
Ah now I see, that makes sense. I thought that may have been it, but it wasn't preciseely clear. Maybe we change the second "the user" to "they" since we already preface with "...users that..."? 

If other folks think it's clear, that's fine! I understand now.

Reply:
> Ah now I see, that makes sense. I thought that may have been it, but it wasn't preciseely clear. Maybe we change the second "the user" to "they" since we already preface with "...users that..."?
> 
> If other folks think it's clear, that's fine! I understand now.


In commit 3d920fc  I changed that sentence to:

>This ensures the rotation control is accessible to people using a screen reader that does not move the mouse or keyboard focus when the user is moving its reading cursor.

Reply:
@annabbott, maybe you didn't see the html file on the files tab because it had lots of lines changed. When the diff is large, github does not load it by default. You have to click a button to show the changed lines in the file.

Reply:
@mcking65's commit 3d920fc solves my issue - my thanks, Matt!